spaces/domains.json

```
@@ -920,4 +920,6 @@
   "vote.chigag.studio": "chigag-studio.eth",
   "dao.space.id": "dao.spaceid.eth",
   "dao.pearl.club": "pearlclubdao.eth"
+  "vote.molens.xyz": "molens.eth"
+
```

A comma is missing at the end of the line above your sub-domain. Please remove also the empty line.
```suggestion
  "dao.pearl.club": "pearlclubdao.eth",
  "vote.molens.xyz": "molens.eth"
```